styles/commit-preview-view.less

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+
+.github-StubItem-git-commit-preview { // TODO Rename class
+  overflow: auto;
+
+  .github-FilePatchView {
+    border-bottom: 1px solid @base-border-color;
+
+    & + .github-FilePatchView {
+      margin-top: @component-padding;
+      border-top: 1px solid @base-border-color;
+    }
+  }
+
+
+  // hack hack hack
+  .github-FilePatchView {
+    height: 500px;
```

Comment:
I think it's not possible to set the height with CSS alone. The reason is that `<atom-text-editor>`s have scrolling inside them and don't expand to whatever height their content is, like normal `<div>`s.

Do `<atom-text-editor>`s have some sort of "auto-height without scrolling" feature? If not we might have to inline each `<atom-text-editor>` with the current `height` of the content. And update it on every change. We probably also lose the performance boost, because now the native `overflow: auto` does the scrolling and not the editor.

Hmmm.. should we consider a single `<atom-text-editor>`? Probably comes with other problems. :grimacing:

Reply:
Ok, nevermind. Turns out there is a `autoHeight` option.